CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+
```

Comment:
the ci should handle changing the CHANGELOG, should be a need to edit it manually.

Reply:
removed